strider/trapi.py

```diff
@@ -50,7 +50,9 @@ def merge_nodes(knodes: list[Node]) -> Node:
     attributes_values = get_from_all(knodes, "attributes")
     if attributes_values:
         output_knode["attributes"] = \
-            filter_none(merge_listify(attributes_values))
+            deduplicate_by(
+                filter_none(merge_listify(attributes_values)),
+                lambda d: tuple(d.items()))
```

Comment:
Technically, attribute `value`s are allowed to have arbitrary JSON types, including lists and dicts. I suspect that this will break in those cases. If we just need a hashable key, maybe `json.dumps(d, sort_keys=True)`?